src/Shared/BundleConstantsRule.php

```diff
@@ -47,7 +49,7 @@ public function apply(AbstractNode $node)
                 $node,
                 [
                     sprintf(
-                        'The value "%s" and the name of constant %s::%s are not equal which violates rule "Only keys no values"',
+                        'The value "%s" and the name of constant %s::%s are not equal which violates rule "Only keys no values" (a prefix separated with ":" is allowed for the value)',
```

Comment:
Shouldn't the prefix mandatory from now on? Maybe you should also add a check for the exactly expected key. The message could then contain exactly what is allowed/expected.

Current const name `FOO_BAR`
Current bundle name `BazBat`

The message: 'The value "not matching value" and the name of constant BazBatConstants::FOO_BAR has not the expected format. Expected "BAZ_BAT:FOO_BAR"'

If the bundle name shouldn't be mandatory you could add 'or FOO_BAR'.

Something like that.